react/src/components/MainLayout/WebUISider.tsx

```diff
@@ -112,31 +130,43 @@ const WebUISider: React.FC<WebUISiderProps> = (props) => {
       ),
       icon: <SessionsIcon style={{ color: token.colorPrimary }} />,
       key: 'job',
+      group: 'workload',
     },
     supportServing && {
       label: <WebUILink to="/serving">{t('webui.menu.Serving')}</WebUILink>,
       icon: <EndpointsIcon style={{ color: token.colorPrimary }} />,
       key: 'serving',
+      group: 'service',
+    },
+    {
+      label: <WebUILink to="/model-store">{t('data.ModelStore')}</WebUILink>,
+      icon: <ModelStoreIcon style={{ color: token.colorPrimary }} />,
+      key: 'mode-store',
```

Comment:
Is it okay to not make it hidden based on the enableModelStore value in config?

Reply:
After Model store  feature became a separated page, it's better to handle using `blocklist` configuration.